test/fidelity/README.md

```diff
@@ -0,0 +1,135 @@
+# Fidelity Testing
+
+It is very important for any 3D model renderer to strive towards producing a
+render that is at once true to the source model, aesthetically pleasing to the
+person viewing the render and as consistent as possible with other commonly
+used renderers.
+
+To that end, the `<model-viewer>` project is pursuing a line of testing that
+compares renders produced by `<model-viewer>` to selected third party reference
+model viewers, with an emphasis on revealing major inconsistencies and rendering
+errors. The goal of this testing is to improve the quality of `<model-viewer>`
+renders over time, as well as to spark discussions with the authors of other
+renderers as we seek to achieve a satisfying balance between correctness,
+beauty and consistency.
+
+## Reference model viewers
+
+There are a lot of model viewers out there. Since one of the main goals of
+`<model-viewer>` is consistency, we will focus on those viewers that have
+significant levels of distribution and/or reputation. Current viewers that we
+are intend to compare to include:
+
+ - [iOS Quick Look](https://developer.apple.com/arkit/gallery/)
+ - [Filament GLTF Viewer](https://github.com/google/filament/blob/master/samples/gltf_viewer.cpp)
```

Comment:
nit: I think the preferred capitalization is "glTF" (even when in a title): https://www.khronos.org/gltf/

Reply:
👍 Will fix